### ISSUE TITLE
Not a bug, but an inconsistency, why sorting products enum is named ProductOrderField and other enums are XSortField ?

### ISSUE THREAD

**Issue by LeOndaz** [mirumee/saleor#6339]
Well, here's an explanation in ts

```
export enum ProductOrderField {
  /** Sort products by name. */
  Name = 'NAME',
  /** Sort products by price. */
  Price = 'PRICE',
  /** Sort products by a minimal price of a product's variant. */
  MinimalPrice = 'MINIMAL_PRICE',
  /** Sort products by update date. */
  Date = 'DATE',
  /** Sort products by type. */
  Type = 'TYPE',
  /** Sort products by publication status. */
  Published = 'PUBLISHED',
  /** Sort products by publication date. */
  PublicationDate = 'PUBLICATION_DATE'
```
and 
```
export enum OrderSortField {
  /** Sort orders by number. */
  Number = 'NUMBER',
  /** Sort orders by creation date. */
  CreationDate = 'CREATION_DATE',
  /** Sort orders by customer. */
  Customer = 'CUSTOMER',
  /** Sort orders by payment. */
  Payment = 'PAYMENT',
  /** Sort orders by fulfillment status. */
  FulfillmentStatus = 'FULFILLMENT_STATUS',
  /** Sort orders by total. */
  Total = 'TOTAL'
}
```

**Comment by LeOndaz** [comment]
And same applies for ProductOrder, XSortingInput ?

**Comment by maarcingebala** [comment]
Thanks for reporting, it should be improved to use the same naming, we'll take care of that - it could be fixed Saleor 3.0.

**Comment by LeOndaz** [comment]
not stale
